src/uct/ib/dc/dc_mlx5_ep.c

```diff
@@ -1305,17 +1346,27 @@ uct_dc_mlx5_iface_dci_do_dcs_pending_tx(ucs_arbiter_t *arbiter,
     int is_only                = ucs_arbiter_elem_is_only(elem);
     ucs_arbiter_cb_result_t res;
 
+    ucs_assert(!uct_dc_mlx5_iface_is_dci_rand(iface));
```

Comment:
can remove it

Reply:
ok, removed

Reply:
why? imo it is valid assert, which makes sure this handler is not used for random policy